src/Command/Branch/BranchRemoteAddCommand.php

```diff
@@ -29,14 +29,26 @@ protected function configure()
             ->setDescription('Adds a remote with url used from adapter')
             ->addArgument(
                 'other_organization',
-                InputArgument::REQUIRED,
+                InputArgument::OPTIONAL,
                 'Organization or username the remote will point to'
             )
+            ->addArgument(
+                'other_repository',
+                InputArgument::OPTIONAL,
+                'Repository-name the remote will point to'
+            )
+            ->addArgument(
+                'remote',
+                InputArgument::OPTIONAL,
+                'Remote name, when not provided the other_organization is used as remote-name'
+            )
             ->setHelp(
                 <<<EOF
 The <info>%command.name%</info> command adds a remote with url used from adapter:
 
-    <info>$ gush %command.name% sstok</info>
+    <info>$ gush %command.name% sstok gush</info>
+
+    , note that this overwrite any existing remote with the same name!
```

Comment:
Changed to `<fg=yellow;options=bold>Warning! Any existing remote with the same name will be overwritten!</>`